components/sup/src/main.rs

```diff
@@ -163,7 +164,7 @@ fn cli<'a, 'b>() -> App<'a, 'b> {
             (@arg ENVIRONMENT: --environment -e +takes_value requires[APPLICATION]
                 "Environment name; [default: not set].")
             (@arg CHANNEL: --channel +takes_value
-                "Receive package updates from the specified release channel")
+                "Receive package updates from the specified release channel [default: stable]")
```

Comment:
I'm pretty sure this was an intermediary step in the first place before we had channels pipe through in the UI of Builder just so we could avoid confusion